pep-0484.txt

```diff
@@ -1297,7 +1383,7 @@ from ``UserId`` where ``int`` is expected. Examples::
 
         num = UserId(5) + 1     # type: int
 
-``NewType`` accepts only one argument that shoud be a proper class,
+``NewType`` accepts only one argument that should be a proper class,
```

Comment:
Hm, while you're at it, can you rephrase this some more? NewType takes 2 args, so this sentence is confusing.